tlatools/org.lamport.tlatools/src/tlc2/tool/impl/SpecProcessor.java

```diff
@@ -123,7 +123,8 @@ public class SpecProcessor implements ValueConstants, ToolGlobals {
     private ModuleNode rootModule; // The root module.
     private Set<OpDefNode> processedDefs;
     private SpecObj specObj;
-    private Defns snapshot;
+    private Defns preConstantSnapshot;
```

Comment:
The changes made to the `SpecProcessor` will also need to be isolated into a separate commit. This is my TODO.